hazelcast-enterprise/maven.functions.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# THIS FILE IS DUPLICATED AND MUST BE KEPT IN SYNC MANUALLY
+# Docker requires any included script to be in the current folder, hence we must duplicate this script for OS and EE
+
+# Prints the latest version in the Maven repository
+#
+# Parameters:
+#   group_id        e.g. com.google.guava
+#   artifact_id     e.g. guava
+#   repository_url  e.g.  https://repo1.maven.org
+#
+# Prints the latest released version of the given artifact in the provided Maven repository
```

Comment:
Should say Jfrog

Reply:
> Should say Jfrog

Do you mean in the documentation of the function? I tried to make it generic as it works for any repo (same for the tests).

Reply:
Yes in the doc. I see what you mean now. When I see Maven repository I am always think about Maven Central but, of course, JFrog is also a Maven repo. I will resolve this